lib/gui/app/pages/finish/controllers/finish.js

```diff
@@ -55,4 +59,22 @@ module.exports = function ($state) {
     analytics.logEvent('Restart', options)
     $state.go('main')
   }
+
+  /**
+   * @summary Format the result errors with newlines
+   * @function
+   * @public
+   *
+   * @returns {String} formatted errors
+   *
+   * @example
+   * const errors = FinishController.formattedErrors()
+   * console.log(errors)
+   */
+  this.formattedErrors = () => {
+    const errors = _.map(_.get(flashState.getFlashResults(), [ 'results', 'errors' ]), (error) => {
+      return `${error.device}: ${error.code}`
```

Comment:
Should this include the error message as well?

Reply:
The EUNPLUGGED error doesn't seem to have one, but I've instead opted for `error.message || error.code`.